src/Pages/ProductDetail/Goods/Goods.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import './Goods.scss';
+
+class Goods extends Component {
+  constructor() {
+    super();
+    this.state = {
+      product_quantity: 1,
+      imgId: 0,
+      isModalOn: false,
+    };
+  }
+
+  goToCart = e => {
+    this.props.history.push('/cart');
+
+    fetch('http://10.58.2.24:8000/orders/cart', {
```

Comment:
api 주소는 src/config.js에 모아서 해주세요!
```suggestion
    fetch(`{API}/orders/cart`, {
```